src/lib/view-manager.js

```diff
@@ -239,7 +239,6 @@ class ViewManager {
      * @param {!View} view The view to set as the foremost view.
      * @param {boolean=} opt_noDispose Whether to dispose the current view.
      *
```

Comment:
could you please remove this empty line as well?

Reply:
Yes, sure